DEFAULT_CONFIG.json5

```diff
@@ -337,8 +330,39 @@
         known_keys_file: null,
       },
     },
+    // acl: {
```

Comment:
I don't think this `acl` configuratino block should be under `transport`. IMHO it should be at the same level of `downsampling`.